v5_MIGRATION_GUIDE.md

```diff
@@ -409,35 +409,36 @@ do not need to be instantiated in `OnCreate`.
 ```diff
 class MyActivity : FragmentActivity() {
 
-+   private lateinit var localPaymentLauncher: localPaymentLauncher
++   private val localPaymentLauncher = localPaymentLauncher()
 -   private lateinit var braintreeClient: BraintreeClient
     private lateinit var localPaymentClient: LocalPaymentClient
 
     @override fun onCreate(savedInstanceState: Bundle?) {
 +       // can initialize clients outside of onCreate if desired
 -       initializeClients()
-+       localPaymentLauncher = LocalPaymentLauncher() { paymentAuthResult ->
-+           localPaymentClient.tokenize(paymentAuthResult) { result -> 
-+                when(result) {
-+                   is LocalPaymentResult.Success -> { /* handle result.nonce */ }
-+                   is LocalPaymentResult.Failure -> { /* handle result.error */ }
-+                   is LocalPaymentResult.Cancel -> { /* handle user canceled */ }
-+               }        
-+           }
-+       }
     }
 
     // ONLY REQUIRED IF YOUR ACTIVITY LAUNCH MODE IS SINGLE_TOP
     override fun onNewIntent(intent: Intent) {
-+       localPaymentLauncher.handleReturnToAppFromBrowser(requireContext(), intent)
++       handleReturnToAppFromBrowser(intent)
     }
-
+    
     // ALL OTHER ACTIVITY LAUNCH MODES 
     override fun onResume() {
-+       localPaymentLauncher.handleReturnToAppFromBrowser(requireContext(), requireActivity().
-+           intent)
++       handleReturnToAppFromBrowser(requireActivity().intent)
+    }
+    
+    fun handleReturnToAppFromBrowser(intent: Intent) {
+       // fetch stored LocalPaymentPendingRequest.Success 
++       fetchPendingRequest()?.let {
```

Comment:
```suggestion
+       fetchPendingRequestFromPersistentStorage()?.let {
```
🧶 : Could also use `getPersistentRequest()`. Maybe I'm being overly cautious, but the take-or-leave for me is to avoid sounding like a network request is being executed.

Reply:
Good call - updated